############
## reactive Planner settings
############
tf_frame: "world"
resolution: 0.2
mesh_resolution: 1.0
visualize_max_z: 5.0
sensor_max_range: 9.0
map_publish_frequency: 1.0
probability_hit: 0.7
probability_miss: 0.4
#treat_unknown_as_occupied: false
visualize_search_space: false
debug: false
sample_orientations: false
orientation_sampling_res: 90.0
tree_progress_display_freq: -1
debug_delay: 0.0
dist_to_goal: 1.5
connection_rad: 0.7
grid_resolution: 0.5
grid_size_x: 20.0
grid_size_y: 20.0
grid_size_z: 0.0
grid_start_x: 0.0
grid_start_y: 0.0
grid_start_z: 0.0
